common/src/main/java/org/conscrypt/Hpke.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License
+ */
+
+package org.conscrypt;
+
+import org.conscrypt.NativeRef.EVP_HPKE_CTX;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+
+/**
+ * Hybrid Public Key Encryption (HPKE) APIs.
+ * <p>
+ * HPKE allows multiple cryptographic operations to be done based on a given setup transaction.
+ * Therefore, the following APIs are stateful after setting up the transaction:
+ * <li>
+ *     <ul>{@link Hpke#open(byte[], byte[])} </ul>
+ *     <ul>{@link Hpke#seal(byte[], byte[])}</ul>
+ *     <ul>{@link Hpke#export(int, byte[])} </ul>
+ * </li>
```

Comment:
I think you may have switched `<ul>` and `<li>`. `<ul>` is the outside one and `<li>` is the inside one.

Reply:
Done. I have swapped them in the latest commit.